djangocms_snippet/models.py

```diff
@@ -20,13 +20,13 @@ class Snippet(models.Model):
     """
     name = models.CharField(
         verbose_name=_('Name'),
-        unique=True,
         max_length=255,
     )
     snippet_grouper = models.ForeignKey(
         SnippetGrouper,
         on_delete=models.PROTECT,
         null=True,
```

Comment:
You shouldn't need the null or the blank here.

It's added initially for the migration here: https://github.com/django-cms/djangocms-snippet/pull/74/files#diff-7975630682285642c8665843c1405c2016fe129a1bc04adc016a3600bbbc3b49R38
It's then removed here: https://github.com/django-cms/djangocms-snippet/pull/74/files#diff-4ca21242eb21cfa24bd2e3f9e4848d967856c53f97e1cd1838f5dc194efcd14fR21

I think if you were to run the migrations again you would create a new migration file.